src/discussions/topics/topic-group/topic/Topic.jsx

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-unused-vars, react/forbid-prop-types */
+import { faQuestionCircle } from '@fortawesome/free-regular-svg-icons';
+import { faComments, faFlag } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useParams } from 'react-router';
+import { Link } from 'react-router-dom';
+import { Routes } from '../../../../data/constants';
+
+function Topic({
+  id,
+  name,
+  subtopics,
+  questions,
+  discussions,
+  flags,
+}) {
+  const { courseId } = useParams();
+
+  return (
+    <Link
+      className="discussion-topic d-flex flex-column list-group-item text-gray-900 text-decoration-none p-2"
+      data-topic-id={id}
+      to={
+        Routes.POSTS.PATH.replace(':courseId', courseId)
+          .replace(':topicId', id)
+      }
+    >
+      <div className="topic-name h6">
```

Comment:
This component can be improved a lot to match the mockups only using bootstrap classes.
Items to be improved: 

- Font sizes, color and decoration 
- Icons sizes and color
- Remove border around each item (`list-group-item`)
- Improve spacing

Ref: https://www.figma.com/proto/LVFnO9oRHYiamoicZuXDAC/Discussion-page?node-id=238%3A1676&scaling=min-zoom

Reply:
@giovannicimolin

> Remove border around each item (list-group-item)

While the styling for the borders might have been stronger than in the mockup, I do see borders in between each item (though they are faint). Should I be removing them entirely?

Reply:
>While the styling for the borders might have been stronger than in the mockup, I do see borders in between each item (though they are faint). 

I guess using my monitor in a warm setting didn't help in getting me to see it. I've switched to `game` mode the lines magically appeared.

>Should I be removing them entirely?

You should keep them.